src/main/java/org/broadinstitute/hellbender/engine/filters/MappingQualityReadFilter.java

```diff
@@ -14,18 +14,28 @@
 public final class MappingQualityReadFilter extends ReadFilter implements Serializable {
```

Comment:
Need to update the class-level docs, which currently say "Keep reads with mapping qualities above a specified threshold."